tests/system/action/meeting/test_import.py

```diff
@@ -583,6 +583,12 @@ def test_replace_ids_and_write_to_datastore(self) -> None:
         self.assert_model_exists("committee/1", {"meeting_ids": [2]})
         self.assert_model_exists("organization/1", {"active_meeting_ids": [1, 2]})
 
+    def test_check_calc_fields(self) -> None:
+        request_data = self.create_request_data({})
+        response = self.request("meeting.import", request_data)
+        self.assert_status_code(response, 200)
+        self.assert_model_exists("user/2", {"meeting_ids": [2]})
```

Comment:
check that the back relation also exists

Reply:
`user/meeting_ids` is a calculated field, there exists no back relation, imo.

Reply:
Ah, okay. I add a test to `meeting/user_ids`.